app/views/waste_carriers_engine/certificates/registration_pdf.html.erb

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <style type='text/css'>
+      * {
+        font-family: Helvetica, Arial, Sans-Serif !important;
+        background: #FFF;
+      }
+      h1 {
+        margin-bottom: 20px;
+      }
+      body {
+        margin: 40px 60px;
+      }
+      table tr td:first-child {
+        width: 220px;
+      }
+      ul, li {
+        list-style-type: none;
+        margin-left: 0;
+        padding-left: 0;
+      }
+      li, td, p {
+        font-size: 18px;
+        line-height: 1.5;
+      }
+      table th {
+        margin-top: 20px;
+        text-align: left;
+        font-size: 22px;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="wrapper">
+
+      <main id="content" role="main" class="group">
+
+        <div class="grid-wrapper">
+        <div class="grid-2-3">
+
+        <!-- Header section -->
+        <header class="page-header group">
+          <div class="hgroup">
+            <h1 class="heading-small"><%=t ".heading" %></h1>
+          </div>
+        </header>
+
+          <table summary="Regulation Authority details" class="certificate_table" role="presentation">
+            <thead>
+              <tr>
+                <th><%=t ".authority" %></th>
+                <td></td>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td class="table-column-text"><%=t ".name" %></td>
+                  <td>
+                    <%= wicked_pdf_image_tag("environment_agency_logo.png", {:alt => "Environment Agency Logo", :border => "0", :height => "39", :width => "129"}) %>
```

Comment:
Guessing this came from the existing template, but can we use the new hash syntax here?